SignItVideosGallery.js

```diff
@@ -17,42 +17,32 @@ var SignItVideosGallery = function ( container ) {
 };
 
 SignItVideosGallery.prototype.refresh = async function ( files ) {
-	var BetterBanana = await browser.storage.local.get( 'bananaInStore' );
-	var messageStore = await chrome.storage.local.get( 'sourceMap' ); 
-	var sourceMap = new Map(messageStore.sourceMap);
-	var locale = BetterBanana.bananaInStore.locale;
 	var banana = {
-    i18n: (msg, url, speaker,index, total) => {
-		let string = sourceMap.get(locale)[msg];
-		let Speaker = `<a href=${url} target="_blank">${speaker} </a>`;
-		let patterns = ["{{link|$1|$2}}", "$3", "$4"];
-		let replacements = [Speaker, index, total];
-
-		patterns.forEach((pattern,index)=>{
-			string = string.replace(pattern, replacements[index]);
-		})
-
-		return string;
+    i18n: async (msg, ...arg) => {
+      return await chrome.runtime.sendMessage({
+        command: "bananai18n",
+        arg: [msg, arg],
+      });
     },
   };
 	console.log("#20 files ",files )
 	console.log("Expected: [{ filename: url, speaker: ... },...]")	
 	var i;
 	files = files || [];
 	this.$videos = [];
-
+	
 	this.$videoContainer.empty();
 	this.currentIndex = 0;
-
 	for ( i = 0; i < files.length; i++ ) {
 		filename = files[ i ].filename,
 		url = `https://commons.wikimedia.org/wiki/File:${ filename.split( '/' ).pop()}`,
 		speaker = files[ i ].speaker,
 		total = files.length;
+		console.log(await banana.i18n("si-panel-videos-gallery-attribution",url, speaker, i+1, total));
```

Comment:
This is not needed in my opinion, might be a good idea to remove this.

Reply:
yea...was doing that to test if the links were being parsed properly. Due to error 502 , videos weren't available and hence had to log them seperately in order to test it. Will remove it in the next PR.